src/components/SortDropdown/types.ts

```diff
@@ -0,0 +1,11 @@
+export enum SortOptionsKeys {
+  APY = 'APY',
+  VOLUME = 'Volume',
+  MARKET_CAP = 'Market Cap',
+  PRICE_CHANGE = 'Price Change',
+}
+
+export type SortOption = {
+  key: SortOptionsKeys
+  label: string
+}
```

Comment:
We can probably remove this type entirely too by using the translations directly from `SortOptionsKeys` values, in a similar way to above. The `SortOptionsKeys` values would need to be updated to be JSON friendly, but that should be fine as I don't think they are used for display anywhere.

Reply:
Done in [1ac600a](https://github.com/shapeshift/web/pull/8006/commits/1ac600aeb95da5729ef78865c837fd0bcec0f777)